User-Help/Platform-Help/Writing-Posts/CodidactAccessibilityHelpArticle.md

```diff
@@ -0,0 +1,255 @@
+
+A bad example would be leaving out the alt text, having overly long alt text, or relying on an external image hosting service:
+
+    ![](https://i.imgur.com/XtjJxjF.jpg) 
+    
+    ![A screenshot of two different sections that you see when editing a post. One says "Edit Comment" with a blank text box, where you can input an edit comment that will show up in the revision history for that post. The other says "Redact", and it has text with an unchecked checkbox. It has the text "Redact original content by hiding the previous revisions from history?" Below those are two buttons. One is blue and says "Save Post in Q&A". The other is gray and says "Cancel".](https://meta.codidact.com/uploads/qwvy25mvolpdjabhknuujzjncbpm)
+
+In the first bad example, there is both no alt text and it relies on an external image service. There is no information presented to anyone who can't see the image, and there's the risk that the image will go dead even for people who can see it.  
+In the second bad example, the alt text is too long. It has information that's not present in the image itself, such as information about the edit comment appearing in the revision history, and describes information that's not relevant to the purpose of the screenshot - which is simply to show what you're presented with when you go to redact a post.
+
+----
+
+<h3 id="Color">Don't rely solely on color</h3>
+
+When your image uses colors to indicate a difference between things - such as on a chart or graph - you should also use a different method of differentiating, such as an icon or different shape. Also avoid using colors that are known to be a problem for colorblind users (such as red/green).
+
+<h3 id="Contrast">Contrast</h3>
+
+Avoid colors that are too close to each other, especially for text on a background color. As a simple way of testing, take a glance at the image in sunlight - can you still make it out?
+
+<h3 id="Animations">Animations</h3>
+
+We don't currently support any way to disable or pause animations in posts, so avoid using animations where possible. In particular, make sure to avoid flashing content (especially anything flashing more than three times a second - don't do that!). Flashing content can cause seizures, and looping animations can be distracting for everyone, but especially for people with some cognitive disabilities.
+
+<h3 id="Text-images">Images of text</h3>
+
+* Avoid images of text.  
+
+  Images of text can't have the text selected, be read by screen readers, indexed by search engines, have the text adjust in a responsive design, or have the font changed. This includes images of code; instead, put the actual code in your post and format it using the dedicated code formatting Markdown.
+
+* If you are quoting from somewhere, use blockquote formatting (which can be done by putting a `>` at the beginning of a paragraph) and cite your source.  
```

Comment:
This bullet and the next one (about code formatting for quotes) seem out of place in the "Images of text" section.  Is there a missing heading, like "Quotes"?

Reply:
It's sort of going off on a tangent, since people using images for quoting text is an issue. I've edited it to be its own subsection.